src/__test__/phantom.spec.js

```diff
@@ -79,15 +79,15 @@ describe('Phantom', () => {
         pp.exit();
     });
 
-    it('#create([], {logLevel: \'debug\'}) change logLevel', () => {
+    it.skip('#create([], {logLevel: \'debug\'}) change logLevel', () => {
         const logLevel = 'error';
 
         let pp = new Phantom([], {logLevel});
         expect(pp.logger.transports.console.level).toEqual(logLevel);
```

Comment:
This test expects `this.logger` to be a winston object which is not the case now. 

It's also not needed since ...